packages/core/src/auth/auth.ts

```diff
@@ -166,6 +168,26 @@ export class Auth implements AuthService, ConnectionManager {
         return this.#ssoCacheWatcher
     }
 
+    public get startUrl(): string | undefined {
+        // Reformat the url to remove any trailing '/' and `#`
+        // e.g. https://view.awsapps.com/start/# will become https://view.awsapps.com/start
```

Comment:
would this comment make sense as the docstring for `reformatStartUrl` instead

Reply:
move the comment to normalizeStartUrl

Reply:
Done